src/resources/closure.lib.d.ts

```diff
@@ -20,6 +20,8 @@ declare namespace ಠ_ಠ.clutz {
    * type position PrivateType is used.
    */
   interface PrivateInterface {}
+
+  interface IObject<KEY1, VALUE> {}
```

Comment:
I was wondering why can't we make this guy ```IObject<KEY1, VALUE> { [key: KEY1]: VALUE}``` and greatly simplify the logic, but TS does not support generic types for index signatures.

http://www.typescriptlang.org/play/#src=interface%20IObject%3CKEY1%2C%20VALUE%3E%20%7B%0A%09%5Bk%3A%20KEY1%5D%20%3A%20VALUE%3B%0A%7D

I will add a comment to remind myself that as I merge the cl into master.